### ISSUE TITLE
Home Unite Us: Update alt text for Home Unite Us image to adhere to WCAG

### ISSUE THREAD

**Issue by plang-psm** [hackforla/website#4012]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, we need to provide clear and descriptive alt text on the `Home Unite Us` page so that we adhere to the Web Content Accessibility Guidelines (WCAG) and we can achieve our mission of inclusivity.

### Action Items
- [x] Change the image alt property value within `_projects/home-unite-us.md`:

**From:**
```
Home Unite Us logo, blue circle with white heart in the center, two hands hugging the logo with "Home Unite Us" above the heart.
```

**To:**
```
Home Unite Us
```

**For PM, Merge Team, or Tech Lead**
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #3866 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- https://www.hackforla.org/projects/home-unite-us

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_projects/home-unite-us.md`
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/gh-pages/_projects/home-unite-us.md
- This issue is part of epic #3866

#### Background information about the Jekyll framework (optional reading)
- [Jekyll Data Files](https://jekyllrb.com/docs/datafiles/)
- [Jekyll Front Matter](https://jekyllrb.com/docs/front-matter/)

#### Information about WCAG and alt text (optional reading)
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

**Comment by chriskong217** [comment]
- On this issue: https://github.com/hackforla/website/issues/4012
The link to the project leads to the 311 data page, which is a different project. 
This is what the issue currently displays:

> Resources/Instructions
> https://www.hackforla.org/projects/311-data

The link should display the correct Home Unite Us link.

**Comment by plang-psm** [comment]
@chriskong217 Nice catch! The link has been updated.

**Comment by erinpattison** [comment]
I. I'll be working on this issue today (3/15). This is my first issue, so I'm following along the contributing.md guide along side this issue. The contributing guide is a little wordy, so bear with me while I figure it out.
II. I should have this completed by tonight (3/15), or tomorrow (3/16) afternoon at the latest.

**Comment by ExperimentsInHonesty** [comment]
Hi @erinpattison It looks like you made a PR on this issue and the reviewers found you had some issues you needed to correct.  

- Issue: https://github.com/hackforla/website/issues/4012
- PR: https://github.com/hackforla/website/pull/4196

Its been a week since they sent you the details. Please reach out for help in the hfla-site slack channel and or go to team meetings to get some support.  

Getting your branching right is the hardest part.  Stick with it.

**Comment by EdASalazar** [comment]
I'm available 10-6 this week. I hope to have it done today or tomorrow.

**Comment by roslynwythe** [comment]
Hi @EdASalazar this issue has been flagged for review because there has been no activity or comments in the past 2 weeks.  Please leave a comment describing your progress, availability, ETA, and any problems or blockers you are encountering, then remove the "2 weeks inactive" label.  Thank you!

**Comment by EdASalazar** [comment]
I think it was already merged. Am I supposed to remove myself as "Assignee" or is it removed after the merge?

**Comment by ExperimentsInHonesty** [comment]
@EdASalazar You did complete the PR but you put the wrong number after the word fixes

in PR #4465 You added

> fixes #4465

Instead of 
`Fixes #4012`
which would of closed this Issue when your pr was done.
